src/main/java/com/google/codeu/servlets/MessageFeedServlet.java

```diff
@@ -0,0 +1,22 @@
+package com.google.codeu.servlets;
+
+import java.io.IOException;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ * Handles fetching all messages for the public feed.
+ */
+@WebServlet("/feed")
+public class MessageFeedServlet extends HttpServlet{
+
+ @Override
+ public void doGet(HttpServletRequest request, HttpServletResponse response)
+   throws IOException {
+
+  response.getOutputStream().println("this will be my message feed");
```

Comment:
This is a small thing but you might consider changing the indentation to match Google's Java style guide?

https://google.github.io/styleguide/javaguide.html#s4.2-block-indentation

Basically, the recommendation is to indent each section using 2 spaces. I think right now you're only using one space?